__tests__/article.test.js

```diff
@@ -151,5 +151,131 @@ describe("05 GET /api/articles", () => {
         });
     });
   });
-  //The only error that should occur here is if the endpoint doesn't exist, which has already been tested during GET /api/topics
+});
+
+describe("08 PATCH /api/articles/:article_id", () => {
+  describe("Functionality", () => {
+    test("When given an object with an inc_votes key, should modify votes by tha value, and return the new article object", () => {
+      const body = { inc_votes: 1 };
+
+      const output = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 101,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toEqual(output);
+        });
+    });
+
+    test("The equivalent row in the database should also return the updated object", () => {
+      const body = { inc_votes: 1 };
+
+      const output = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: new Date("2020-07-09T20:11:00.000Z"),
+        votes: 101,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(200)
+        .then(() => {
+          return db.query("SELECT * FROM articles WHERE article_id = 1");
+        })
+        .then(({ rows }) => {
+          expect(rows[0]).toEqual(output);
+        });
```

Comment:
I don't think this is adding too much more confidence in the endpoint, considering the test before gave us a lot of confidence already but I can see the appeal